content/english/linux-basics/activity-5.md

```diff
@@ -63,4 +82,3 @@ What permissions do the other two values give?
 | 2 | write |
```

Comment:
I don't like the explanation that 7 is a combination of 4, 2, 1. Might be better to just fully enumerate what 3, 5, 6, 7, all mean...